### PR TITLE
make 'cloud network private details' support showing all private info

### PR DESCRIPTION
**Pull request by sgsullivan**
.. this makes it nicer

### CODE REVIEW

File: cmd/cloudNetworkPrivateDetails.go [hunk 1]
```diff
@@ -39,37 +42,53 @@ Applications that communicate internally will frequently use this for both secur
 and cost-savings.
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		uniqIdFlag, _ := cmd.Flags().GetString("uniq-id")
+		allFlag, _ := cmd.Flags().GetBool("all")
```
Comment (dwburke) [inline cmd/cloudNetworkPrivateDetails.go:45]: compared to things like "cloud server status" wouldn't it be more consistent with lw-cli's behaviour to have default be all, then pass uniq-id(s) to show a specific server(s)?

Comment (sgsullivan) [reply cmd/cloudNetworkPrivateDetails.go]: thats how its acting

```ssullivan@data ~/golang/src/github.com/liquidweb/liquidweb-cli $ lw cloud network private details
Cloud Server [YN3WSA] is not attached to a Private Network
Cloud Server [AFP9BK] is not attached to a Private Network
Cloud Server [01KQNV] is not attached to a Private Network
ssullivan@data ~/golang/src/github.com/liquidweb/liquidweb-cli $
```

That said, maybe thats not very clear by having that `all` flag. I removed the useless flag a6f141e443ff93d3b217acfa555306c38ed7c13c

Comment (sgsullivan) [reply cmd/cloudNetworkPrivateDetails.go]: thats how its acting

```
ssullivan@data ~/golang/src/github.com/liquidweb/liquidweb-cli $ lw cloud network private details
Cloud Server [YN3WSA] is not attached to a Private Network
Cloud Server [AFP9BK] is not attached to a Private Network
Cloud Server [01KQNV] is not attached to a Private Network
ssullivan@data ~/golang/src/github.com/liquidweb/liquidweb-cli $
```

That said, maybe thats not very clear by having that `all` flag. I removed the useless flag a6f141e443ff93d3b217acfa555306c38ed7c13c

**Review by dwburke — APPROVED** [review]

File: cmd/cloudNetworkPrivateDetails.go (hunk 1 above)
Comment (dwburke) [reply cmd/cloudNetworkPrivateDetails.go]: 👍

**Pull request merged**